Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/RideBooking/API/MeterRide.yaml

```
@@ -0,0 +1,30 @@
+imports:
+  Ride: Domain.Types.Ride
+  RateCardItem : API.Types.UI.DriverOnboardingV2
```

YAMLlint (1.35.1)

[warning] too many spaces before colon

(colons)